core/pkg/artifacts/saver.go

```diff
@@ -138,6 +138,41 @@ func (as *ArtifactSaver) createManifest(
 	return response.GetCreateArtifactManifest().ArtifactManifest, nil
 }
 
+func (as *ArtifactSaver) updateManifest(
+	artifactManifestId string, manifestDigest string,
+) (attrs gql.UpdateArtifactManifestUpdateArtifactManifestUpdateArtifactManifestPayloadArtifactManifest, rerr error) {
```

Comment:
It might be nice to create a type alias for this absurd generated name, especially if if's more informative